### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by sehilyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ At present this list is very biased towards things [@petulla](http://github.com/
 
 ## Color
 
-- [Affective Colour Palettes in Visualization](https://research.tableau.com/paper/affective-colour-palettes-visualization-poster) - how can different color palletes convey different emotions?
+- [Affective Colour Palettes in Visualization](https://research.tableau.com/paper/affective-colour-palettes-visualization-poster) - how can different color paletes convey different emotions?
```
Comment (petulla) [inline README.md]: Should be: "palette"

**Review by petulla — CHANGES_REQUESTED** [review]
Could you fix that one typo, then we'll merge? Thanks for this.

**Comment by sehilyi** [comment]
Fixed the typo according to your review :)

**Comment by mathisonian** [comment]
Thanks @sehilyi!

**Pull request merged**